client/src/components/overview/Info/Info-helper.jsx/star-helper.jsx

```diff
@@ -0,0 +1,19 @@
+const averageRating = (reviewResults, changeState) => {
+  let ratings = 0;
+  let totalRatings = 0;
+
+  if (reviewResults && reviewResults.length !== 0) {
+    for (let i = 0; i < reviewResults.length; i += 1) {
+      if (reviewResults[i].rating !== undefined) {
+        ratings += reviewResults[i].rating;
```

Comment:
Reminder that the reviewResults array is not currently taking into account all ratings